docs/data-guidelines.md

```diff
@@ -282,3 +282,23 @@ A single-engine feature's `experimental` status may expire and switch to `false`
 | An API supported in Firefox, released three years ago.                      | No           |
 
 This guideline was proposed in [#6905](https://github.com/mdn/browser-compat-data/issues/6905) and adopted in [#9933](https://github.com/mdn/browser-compat-data/pull/9933).
+
+## Arguments and argument object features
+
+Sometimes it's useful to represent support for specific arguments of a function or method, as a subfeature of the function itself. To record data about whether a specific argument (parameter) is supported by a function or method, use the following naming conventions:
+
+- For named arguments, use a subfeature named `argname_argument` with description text `<code>argname</code> argument`. Where _argname_ is the name of the argument as it appears on the corresponding function's MDN page (or specification, if no MDN page is available).
```

Comment:
I think the most common practice right now is to name the feature after the argument alone (e.g., for `hello(firstName, familyName)`, the feature would be `firstName`). I was inspired to follow a cue from the `_event` features here, but I'm not committed to it. (I do think that we should adopt a common description text though.)